models/tmp/stg_reddit_ads__account_report_tmp.sql

```diff
@@ -1 +1,4 @@
-select * from {{ var('account_report') }}
+{{ config(enabled=var('ad_reporting__reddit_ads_enabled', True)) }}
+
+select * 
+from {{ var('account_report') }}
```

Comment:
Small whitespace edit suggestion.

Please apply the same changes to the other tmp models.

Reply:
updated